ProcessLib/HeatTransportBHE/BHE/CreateFlowAndTemperatureControl.cpp

```diff
@@ -86,6 +87,31 @@ FlowAndTemperatureControl createFlowAndTemperatureControl(
                                       refrigerant.specific_heat_capacity,
                                       refrigerant.density};
     }
+
+    if (type == "BuildingPowerCurveConstantFlow")
```

Comment:
Either the class name should be `BuildingPowerCurveConstantFlow` or the type here should be `BuildingPowerCurve`.